Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM ruby:3.1.2
+FROM ruby:3.1.2-alpine
 
 # set up workdir
 RUN mkdir /circuitverse
 WORKDIR /circuitverse
 
 # install dependencies
-RUN apt-get update -qq && apt-get install -y imagemagick shared-mime-info && apt-get clean
+RUN apk update -qq && apk add imagemagick shared-mime-info postgresql-dev openssl
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
- && apt-get update && apt-get install -y nodejs && rm -rf /var/lib/apt/lists/* \
- && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
- && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
- && apt-get update && apt-get install -y yarn && rm -rf /var/lib/apt/lists/*
+RUN apk --no-cache add nodejs yarn --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community \
```

Comment:
AFAIK this will always install latest node version, can we get more control on this or use latest LTS version here. Because latest version node can have incompatibilities with the packages we use

Reply:
The alpine repository only has the edge channel releases it seems
https://pkgs.alpinelinux.org/packages?name=yarn

Reply:
We may have to find better alternatives, merging this PR for now